### PR TITLE
Dev additional words

### PR DESCRIPTION
**Pull request by goldyukol**

### CODE REVIEW

**Comment by KateBlazhko** [comment]
Check with lint or cherk update develop branch - a lot of errors

**Review by kubarenatioN — APPROVED** [review]
Good work

File: static/src/components/AdditionalWords/AdditionalWords.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
+import Words from "../api/Words";
+import Control from "../common/control";
+import Input from "../common/Input";
+import Logging from "../login/Logging";
+import ErrorNewWord from "../utils/ErrorNewWord";
+import { FORM_INPUTS_FILE, FORM_INPUTS_TEXT } from "./formInputs";
```
Comment (KateBlazhko) [inline static/src/components/AdditionalWords/AdditionalWords.ts:6]: Correct path for module in this string

It should be "./FormInputs", not "./formInputs"

Comment (goldyukol) [reply static/src/components/AdditionalWords/AdditionalWords.ts]: Fixed. Done

**Pull request closed without merging**